packages/generator-single-spa/src/svelte/templates/src/App.svelte

```diff
@@ -1,5 +1,5 @@
-<script>
-  export let name;
+<script lang="ts">
```

Comment:
For create-single-spa's react implementation, typescript is an option passed to the generator, with EJS template for adding typescript syntax. We should add the same for svelte.